.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+<!-- **This issue tracker is a tool to address bugs in wtforms itself.
```

Comment:
I'd remove these header comments. GitHub can now show multiple templates, and some of them can be links to other places instead. https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository The comments also show up in the hover cards instead of the first visible content, which makes them less useful. In my experience people ask questions here anyway, so I have essentially this comment as a canned response that I close with.

Reply:
Removed :)